_posts/2016-05-16-rust-at-one-year.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title: "One year of Rust"
+author: Aaron Turon
+description: "Rust's trajectory one year after 1.0"
+---
+
+Rust is a language that gives you:
+
+- uncompromising performance and control;
+- prevention of entire categories of bugs, including [classic concurrency pitfalls];
+- ergonomics that often rival languages like [Python] and [Ruby].
+
+It's a language for writing highly reliable, screamingly fast software&mdash;and
+having fun doing it.
+
+[classic concurrency pitfalls]: http://blog.rust-lang.org/2015/04/10/Fearless-Concurrency.html
+[Python]: http://lucumr.pocoo.org/2015/5/27/rust-for-pythonistas/
+[Ruby]: http://diesel.rs/
+
+And yesterday, Rust turned one year old.
+
+### Rust in numbers
+
+A lot has happened in the last 365 days:
+
+- 11,894 [commits] by 702 contributors added to the core repository;
+- 88 [RFCs] merged;
+- 18 compiler targets introduced;
+- 9 releases shipped;
+- 1 year of [stability delivered].
+
+On an **average week** this year, the Rust community merged two RFCs and
+published 53 brand new [crates]. Not a single day went by without at least one
+new Rust library hitting the central package manager. And Rust topped the
+"[most loved] language" in this year's StackOverflow survey.
+
+> Speaking of numbers: we recently launched a [survey] of our own, and want to
```

Comment:
I almost skipped over this because it looked like a quote/comment in the github preview.  Does this get rendered as a call-out in the real blog?